src/compiler/transformers/component-lazy/lazy-element-getter.ts

```diff
@@ -21,8 +21,12 @@ export const addLazyElementGetter = (
         cmp.elementRef,
         [],
         undefined,
-        ts.createBlock([
-          ts.createReturn(ts.createCall(ts.factory.createIdentifier(GET_ELEMENT), undefined, [ts.createThis()])),
+        ts.factory.createBlock([
+          ts.createReturn(
```

Comment:
Could/should this be `ts.factory.createReturnStatement`?

Reply:
I could pull that one in one this PR too, my strategy is to start with one file and change only the methods that appear in that single file, but change every use of them across the whole project, so I didn't change this here because `ts.createReturn` doesn't appear in `src/compiler/transformers/component-native/native-connected-callback.ts`

maybe tomorrow I'll just expand this PR to get as many of them as possible...stay tuned 😄

Reply:
Ahhh I see! No worries - I misunderstood the 'branch out' strategy at first